config/routes.js

```diff
@@ -105,6 +108,9 @@ router.get('/avatars', avatars.allJSON);
 router.get('/play', index.play);
 router.get('/', index.render);
 
+// dashboard
```

Comment:
Delete if not being used. There's no need to keep a code that is commented out in the file